crates/ruma-events/src/room/aliases.rs

```diff
@@ -58,6 +59,13 @@ pub struct RedactedAliasesEventContent {
     pub aliases: Option<Vec<RoomAliasId>>,
 }
 
+impl RedactedAliasesEventContent {
+    /// Create a `RedactedAliasesEventContent` with the given aliases.
```

Comment:
I think it would be nice to have it clearer that this is meaningful for v1-v5 only. Maybe rename to `new_v1` and also add `new_v6` that's the same as the `Default` impl?